packages/website/components/Footer.tsx

```diff
@@ -25,47 +26,124 @@ export const Footer = () => {
   return (
     <footer css={styles.footer}>
```

Comment:
The footer in the designs is white and has a border-top

but since you are not changing any CSS in this file, I wanted to ask if you are planing to do these changes in another PR... are you?

Reply:
I think Dominik said something about keeping the gray, I was not sure if we want to do it. maybe let's follow up with him?